src/tests/test_simulation.py

```diff
@@ -58,3 +59,18 @@ def test_error_tracking_nonexistent(sample_pop):
     with pytest.raises(SimulationException,
                        match="Unrecognised tracked attribute"):
         run_simulation(sample_pop, config)
+
+
+def test_death_occurs():
+    """Check that the number of people alive always decreases."""
+    # FIXME This will not necessarily be true once we add in births
+    population = Population(10000, date.today())
+    start = population.date
+    step = timedelta(days=30)
+    end = start + 100 * step
+    config = SimulationConfig(start, end, step, ['num_alive'])
+    pop, results = run_simulation(population, config)
```

Comment:
that and the line above would need to change once you merged the code from my branch. E.g. run_simulation takes only one config (the simulation config) and Population is a class member of SimulationHandler